sippy-ng/src/releases/ReleaseOverview.js

```diff
@@ -187,9 +205,135 @@ export default function ReleaseOverview(props) {
+                  <Link
+                    to={withSort(
+                      `/jobs/${props.release}`,
+                      'current_pass_percentage',
+                      'asc'
+                    )}
+                  >
+                    Job histogram
+                  </Link>
+                  <Tooltip
+                    title={
+                      'Histogram of job pass rates. Bucketed by current period pass percentage. ' +
+                      'Tech preview and never-stable jobs are excluded. The solid line indicates the current ' +
+                      "period's mean, and the dashed line is the previous period."
+                    }
+                  >
+                    <InfoIcon />
+                  </Tooltip>
+                </Typography>
+                <Histogram
+                  data={data.current_statistics.histogram}
+                  current_mean={data.current_statistics.mean}
+                  previous_mean={data.previous_statistics.mean}
+                  release={props.release}
+                />
+                <div align="center">
+                  <span style={{ marginRight: 10 }}>
+                    1Q: {data.current_statistics.quartiles[0].toFixed(0)}%
+                  </span>
+                  <span style={{ marginRight: 10 }}>
+                    2Q: {data.current_statistics.quartiles[1].toFixed(0)}%
+                  </span>
+                  <span style={{ marginRight: 10 }}>
+                    3Q: {data.current_statistics.quartiles[2].toFixed(0)}%
+                  </span>
+                  <span style={{ marginRight: 10 }}>
+                    SD: {data.current_statistics.standard_deviation.toFixed(2)}
+                  </span>
+                </div>
+              </Card>
+            </Grid>
+
+            <Grid item md={7}>
+              <Card elevation={5} style={{ padding: 20, height: '100%' }}>
+                <Typography variant="h6">
+                  <Link
+                    to={`/jobs/${
+                      props.release
+                    }/analysis?filters=${encodeURIComponent(
+                      JSON.stringify({
+                        items: [
+                          ...hourFilter(dayOffset),
+                          BOOKMARKS.NO_TECHPREVIEW,
+                          BOOKMARKS.NO_NEVER_STABLE,
+                        ],
+                        linkOperator: 'and',
+                      })
+                    )}&period=hour}`}
+                  >
+                    {dayOffset === 1
+                      ? 'Last 24 hours'
+                      : `${dayOffset * 24} hours ago`}
+                  </Link>
+                  <Tooltip
+                    title={
+                      'This chart shows a 24 hour period of  job runs, excluding never-stable and tech preview. ' +
+                      'The most recent hour will generally look more successful until all jobs finish running. ' +
```

Comment:
we can't excluded running jobs?

Reply:
Sorry this is worded a little confusingly.  The chart will show running jobs in gray so it's clear we don't have results for those, but the problem is mostly because of jobs that haven't even started.

At the top of the hour the quickest running jobs (e.g. promote) show up green, but later on when everything has run, it settles down and looks more typical. Does that make sense?

Reply:
yeah that makes sense.

```suggestion
                      'The most recent hour will generally look more successful until all jobs finish running because it will be heavily biased towards the shortest jobs(more likely to have already completed) which also tend to be the most reliable. ' +
```

Reply:
Thanks! That's much better